pages/nfttest/mint/_iscnId.vue

```diff
@@ -74,28 +178,26 @@ export default class NFTTestMintPage extends Vue {
 
   async doAction() {
     /* eslint-disable no-fallthrough */
-    switch(this.state) {
+    switch (this.state) {
       case 'create':
-        await this.createNftClass();
-      case 'mint':
-        await this.sendNFT();
-        await this.postMintInfo();
-        await this.getMintInfo();
-        break;
+        this.isLoading = true
+        await this.createNftClass()
+        await this.sendNFT()
+        await this.postMintInfo()
+        await this.getMintInfo()
+        this.isLoading = false
+        break
       case 'done':
-        this.$router.push(this.localeLocation(
-          { name: 'nfttest-button-iscnId', params: { iscnId: this.iscnId } },
-        )!);
+        window.location.href = 'https://github.com/likecoin/likecoin-button-sdk'
       default:
     }
     /* eslint-enable no-fallthrough */
   }
 
   async getISCNInfo() {
-    const res = await this.fetchISCNById(this.iscnId);
+    const res = await this.fetchISCNById(this.iscnId)
     if (res) {
-      console.log(res);
-      [this.iscnData] = res.records;
+      ;[this.iscnData] = res.records
```

Comment:
Not sure if it should be `this.iscnData = res.records[0].data` instead, please check